bot/admin/admin.py

```diff
@@ -41,14 +41,14 @@ async def echo(self, ctx: commands.Context, channel: Optional[discord.TextChanne
 
     @commands.group(name='embed', hidden=True, invoke_without_command=True)
     @command_log
-    async def embed(self, ctx: commands.Context, channel: discord.TextChannel, color: discord.Colour, *, text: str):
+    async def embed(self, _ctx: commands.Context, channel: discord.TextChannel, color: discord.Colour, *, text: str):
```

Comment:
Does mypy not say anything anymore. 
I actually thought about something like: 
`async def embed(self, _: commands.Context, channel: discord.TextChannel, color: discord.Colour, *, text: str):`

Reply:
Nope, MyPy is fully ok with this and doesn't whine about that this argument isn't being used anymore.